### PR TITLE
PSMDB-1221 Release notes 4.4.19-19

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release_notes/4.4.19-19.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by sandraromanchenko — APPROVED** [review]

File: docs/release_notes/4.4.19-19.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+* [PSMDB-1148](https://jira.percona.com/browse/PSMDB-1148): Improve the master key rotation when using a single master key for data-at-rest encryption with Vault in the entire deployment
+
+
+## Bugs Fixed
+
+* [PSMDB-1201](https://jira.percona.com/browse/PSMDB-1201): Improved the error message if the attempt to save an encryption key to a KMIP server failed
+* [PSMDB-1203](https://jira.percona.com/browse/PSMDB-1203): Gracefully terminate `mongod` if the master encryption key can’t be saved to a KMIP server
+* [PSMDB-1204](https://jira.percona.com/browse/PSMDB-1204): Fixed the handling of attributes list for LDAP authentication with OpenLDAP during the user to DN mapping stage
+* [PSMDB-1229](https://jira.percona.com/browse/PSMDB-1229): Fixed the issue with PSMDB exiting after restart by properly reading of the master key identifier
```
Comment (sandraromanchenko) [inline docs/release_notes/4.4.19-19.md]: Let's remove PSMDB-1229 from here as well)

**Review by igorsol — APPROVED** [review]
LGTM

File: docs/release_notes/4.4.19-19.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+|----------------- | ---------------- | 
+| **Installation** | [Installing Percona Server for MongoDB](../install/index.md)|
+
+Percona Server for MongoDB 4.4.19-19 is a source available, highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.19 Community Edition enhanced [with enterprise-grade features](../comparison.md). 
+It supports MongoDB 4.4.19 protocols and drivers.
+
+## Release Highlights
+
+* The support for authentication using [AWS IAM](../aws-iam.md) enables you to natively integrate Percona Server for MongoDB with AWS services, increase security of your infrastructure by setting up password-less authentication and offload your DBAs from managing different sets of secrets. This is the [technical preview feature](../glossary.md#technical-preview-feature)
```
Comment (janwieremjewicz) [inline docs/release_notes/4.4.19-19.md]: Sounds weird to me "this is the ..." should it not be "this is a..." or maybe "This feature is currently in ..."

Comment (nastena1606) [reply docs/release_notes/4.4.19-19.md]: Changed to "a tech preview'

**Pull request merged**